### DISCUSSION TITLE
New maze-themed course

### DISCUSSION THREAD

**Discussion by KazzaRazza** [jaylawl/Jaygrounds#10] [Ideas]
I think it would be a cool idea to have a new course built within a maze, mess with some minds while trying to play a little golf :P

**Comment by TheMysterys** [comment]
Oh this sounds interesting. Not sure how you would get trapped though since you would just follow the course the way you came in to get out.

**Reply by jaylawl** [reply]
i see some potential with it being variable... like an ever changing maze

**Reply by KazzaRazza** [reply]
You can get trapped/turned around while trying to get to the next tee, more then anything I think the layout would look very cool and different

**Reply by TheMysterys** [reply]
Oh yeah that sounds fun

**Reply by jaylawl** [reply]
not too sure about the getting-trapped part because it has to be playable at any time 🤔

**Reply by KazzaRazza** [reply]
That's why I said that I think the maze type layout in itself would look cool and not mandatory to have the getting lost part implemented into it. More so just a new course design
I mean I guess if you want the normal maze experience there could be a maze mini game on the side for some type of prizes but I would also like to just see it as a course :)

**Reply by jaylawl** [reply]
but then again you kinda have to get lost due to it being a maze 🤔 but not due to virtue of getting physically stuck but by it changing? we'd need to develop a concept on all this
